[36384] Add job state time limit actions to batch queue (#36658)
hashicorp/terraform-provider-aws: website/docs/r/batch_job_queue.html.markdown

* `compute_environment` - (Required) The Amazon Resource Name (ARN) of the compute environment.
* `order` - (Required) The order of the compute environment. Compute environments are tried in ascending order. For example, if two compute environments are associated with a job queue, the compute environment with a lower order integer value is tried for job placement first.

### job_state_time_limit_action

> * `action` - (Required) The action to take when a job is at the head of the job queue in the specified state for the specified period of time.

Reviewer: ```suggestion
* `action` - (Required) The action to take when a job is at the head of the job queue in the specified state for the specified period of time. Valid values include `"CANCEL"`
```

> * `max_time_seconds` - (Required) The approximate amount of time, in seconds, that must pass with the job in the specified state before the action is taken.

Reviewer: ```suggestion
    * `job_state_time_limit_action.#.max_time_seconds` - The approximate amount of time, in seconds, that must pass with the job in the specified state before the action is taken. Valid values include integers between `600` & `86400`
```

* `reason` - (Required) The reason to log for the action being taken.

> * `state` - (Required) The state of the job needed to trigger the action.

Reviewer: ```suggestion
* `state` - (Required) The state of the job needed to trigger the action. Valid values include `"RUNNABLE"`.
```
